src/monattack.cpp

```diff
@@ -1115,6 +1163,9 @@ void mattack::fungus_sprout(monster *z, int index)
 
 void mattack::fungus_fortify(monster *z, int index)
 {
+    if( z->friendly ) {
+        return; // TODO: handle friendly monsters
```

Comment:
The giant tendril should snipe hostiles and there's no need for a Mycus injection, but the hedgerows still need maintained.